From a93cd11d9c513a6751b59c1c9a72f12d4bb6e160 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 28 Jan 2020 10:17:23 +0200
Subject: [PATCH] [SW-0] goya2: fix offset to soc config space

offset of configuration space of soc in ARC processor
is on the 15th memory region so adjusted offset accordingly

Change-Id: I96a9b91640ad441475326d6acff3fefd2de4fd53
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/include/preboot.h b/include/preboot.h
index 293333f..889a0d0 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -13,7 +13,7 @@
 
 #ifdef __arc__
 #define HL_SPI_BASE				0x1A000000
-#define HL_CFG_BASE				0x1C000000
+#define HL_CFG_BASE				0xFC000000
 #define SDRAM_BASE				0x10000000
 #define UBOOT_TEXT_BASE				0x10100000
 #endif
